.github/workflows/pipeline.yml

```
@@ -0,0 +1,78 @@
+            
+            # Instala dependencias (omitiendo las de desarrollo para un entorno más limpio)
+            npm install --omit=dev
+            
+            # Aplica cualquier nueva migración a la base de datos
+            npx prisma migrate deploy
+            
+            # Genera el build de producción en el servidor
+            npm run build
+            
+            # Reinicia la aplicación usando PM2 para aplicar los cambios sin downtime
+            pm2 restart lti-backend
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)